.gitignore

```diff
@@ -1,9 +1,5 @@
 node_modules
 
-/server-cert.pem
-
-/server-key.pem
-
```

Comment:
You should never check keys into source control as everyone would able to spoof you